### DISCUSSION TITLE
associate-android-app-links-with-your-subdomain-using-github-pages-cloudflare/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#48] [Comments]
# associate-android-app-links-with-your-subdomain-using-github-pages-cloudflare/

As part of my APOD wallpaper rewrite, I wanted to support universal links / app links. These allow users with the app installed to click a link like apod.jakelee.co.uk/day/2020-01-01 and be taken directly to that APOD within the app!

https://blog.jakelee.co.uk/associate-android-app-links-with-your-subdomain-using-github-pages-cloudflare/

**Comment by grubengraeber** [comment]
That was really helpful, thanks a lot!!
